src/workerd/jsg/resource.h

```diff
@@ -191,6 +194,141 @@ template <typename TypeWrapper,
     typename Indexes>
 struct MethodCallback;
 
+// Specialization for const instance methods
```

Comment:
Why do we need this? We haven't needed const API methods up until now, can we go on just not using them and avoid all this extra code?